PokeDemo/Helpers/Internet.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  File.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+
+import UIKit
+import Foundation
+import SystemConfiguration
+
+public protocol InternetManager {
```

Comment:
We don't need this. Remove InternetManager. If we need this specific use case, we'll introduce it later. As of now if there is no internet, url connection will send time out error response.

Reply:
Covered in 7db07cefc3d5401a459807214b87c0e8202a8f7e